doc/deployment-integration/local-deployment/LOCAL.MD

```diff
@@ -3,6 +3,19 @@
 In order to support development and exploration of the FIWARE Data Space Connector, a "Minimal Viable Dataspace" is
 provided as part of this repo.
 
+## Requirements
+
+The local deployment tries to be as decoupled from the host system as possible to reduce the amount of requirements but stills needs the following programs:
+
```

Comment:
If we really need to provide that here, we also should set the minimum versions for docker and maven.
As an alternative (and much better maintainable) approach, why not just link to the github action? It clearly defines the environment and ensures that the tests(and therefor the local deployment) can be executed in.